ocw_import/api_test.py

```diff
@@ -545,3 +549,84 @@ def test_update_ocw2hugo_course_content(
             ).count()
             == (1 if create_new_content else 0)
         )
+
+
+class TestUpdateContentFromS3Data:
```

Comment:
@mbertrand Let me know if you have any suggestions about this test. 

I wanted to be fairly explicit with assertions (e.g., assert against literals not computed expected dictionaries) but also wanted to not copy paste bunch of stuff between tests, hence the shared setup is in this class.

The `update_ocw2hugo_content` function which is more directly used by the management commands still has no tests, but at least this is something :/